### ISSUE TITLE
Run Graphql-voyager broken

### ISSUE THREAD

**Issue by redbass** [APIs-guru/graphql-voyager#84]
I try to run locally Graphql-Voyager, but the graph is not loading, due

> Failed to load http://cdn.jsdelivr.net/npm/graphql-voyager@1.0.0-rc.15/dist/voyager.worker.js: Redirect from 'http://cdn.jsdelivr.net/npm/graphql-voyager@1.0.0-rc.15/dist/voyager.worker.js' to 'https://cdn.jsdelivr.net/npm/graphql-voyager@1.0.0-rc.15/dist/voyager.worker.js' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://localhost:8080' is therefore not allowed access. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

The issue seems related to a change in `https://www.jsdelivr.com/` 
   
> jsDelivr is becoming HTTPS-only. All HTTP requests are redirected to HTTPS. HSTS is going to be enabled in a few days.

The current code in `middleware/render-voyager-page` is importing the dependencies in this way:
`<script src=\"//cdn.jsdelivr.net/fetch/2.0.1/fetch.min.js\">`

Instead now have to be :
`<script src=\"https://cdn.jsdelivr.net/fetch/2.0.1/fetch.min.js\">`

**Comment by IvanGoncharov** [comment]
@redbass Thanks for reporting 👍 
I will try to publish a new version in the next few days.

**Comment by redbass** [comment]
[fix.patch.txt](https://github.com/APIs-guru/graphql-voyager/files/2345145/fix.patch.txt)
Here the fix

**Comment by IvanGoncharov** [comment]
@redbass Thanks 👍 
Do you want to open a PR?

Note: currently we are working on `2.0.0` so current master is not ready to be published so I need a few days to clean it up.

**Comment by redbass** [comment]
ok I will do that

**Comment by redbass** [comment]
https://github.com/APIs-guru/graphql-voyager/pull/85

**Comment by Harmonickey** [comment]
We are eagerly awaiting your 2.0.0 so that we can grab from npm, mostly to fix this jsdelivr issue!  Thanks so much for the good work!

**Comment by DblK** [comment]
Thank you for the PR, but for middleware `middleware/render-voyager-page` would it be possible to use a local file instead of the one from the cdn?
That way it could work offline locally.

**Comment by IvanGoncharov** [comment]
@redbass @Harmonickey @DblK We still in process of heavy refactoring with an aim to finally  release `1.0.0`
But this bug is extremely critical so I released `v1.0.0-rc.20` 📦 with the fix
Can you please test it to ensure that we didn't break anything?

**Comment by marc-rutkowski** [comment]
Hello @IvanGoncharov and thanks for all the good work!
Just testing the rc20 now, and it seems that the middleware folder is missing into dist :(
Any idea?

**Comment by DblK** [comment]
I just tested with `rc.21` and I got this:
﻿`
GET http://localhost:8017/voyager.worker.js 404 (Not Found) voyager.min.js:78 
`

**Comment by alekseyBatuhtin** [comment]
@DblK the same thing.

**Comment by IvanGoncharov** [comment]
@DblK @alekseyBatuhtin Thanks for reporting. I'm looking into it.
Sorry for the inconvenience, I created a test setup for middleware and now I can test it myself.

**Comment by IvanGoncharov** [comment]
@redbass @Harmonickey @DblK @marc-rutkowski @alekseyBatuhtin 
I released `rc-23` 📦 and it works in my local setup.
Can you please test it?

**Comment by Harmonickey** [comment]
Looks good, it loads for me now!  Was clicking around and saw this error though when I clicked the query dropdown button, we can create a new ticket or you can address now, up to you.

![image](https://user-images.githubusercontent.com/3519788/45366155-cee1c780-b5a3-11e8-8cc3-da12d6026300.png)

![image](https://user-images.githubusercontent.com/3519788/45366170-d7d29900-b5a3-11e8-85ae-438a9388abc4.png)

**Comment by marc-rutkowski** [comment]
Works for me too! Thanks for the fix!
And I see the same error on the rootType dropdown...

**Comment by IvanGoncharov** [comment]
@Harmonickey I forgot to update React version in middleware.
Fixed in `rc.24`.

**Comment by IvanGoncharov** [comment]
🤦‍♂️ Fixed stupid mistake, and released `rc.25`.
I tested locally and it looks like now everything works as expected.

**Comment by Harmonickey** [comment]
@IvanGoncharov confirmed on my side, looks good!  awesome work on this library!

**Comment by RomanGotsiy** [comment]
`v1.0.0-rc.20` -> `v1.0.0-rc.25` to fix a simple url issue

We've nailed it 🙈😂! (special thanks go to @IvanGoncharov 👏)

![meme](https://78.media.tumblr.com/e68ad3308e9701c10ca6aebe56008be5/tumblr_inline_p6nhl0n3qv1vvskl5_500.jpg)
